command-line/lib/Options/Applicative/Script.hs

```diff
@@ -0,0 +1,133 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+{-# OPTIONS_HADDOCK hide #-}
+
+module Options.Applicative.Script
```

Comment:
Sorry for the churn - I meant to also say that the ScriptParser module ought to go in the core cardano-address package.
And `scriptArg` can go back to the `Command.Script.Hash` module.
That will make the multisig script mini-language self-contained and re-usable.

Reply:
Created Cardano.ScriptParser, also changed module name : Cardano.Multisig -> Cardano.Script. Left `scriptArg` in Options.Applicative.Script to comply with convention already adopted for other bits (just not to be outlier here). Everything done here -> https://github.com/input-output-hk/cardano-addresses/pull/76/commits/112721d68284f0b580e4f96868761380e67ffe1f